src/worldcereal/openeo/inference.py

```diff
@@ -0,0 +1,67 @@
+"""Model inference on Presto feature for binary classication"""
+
+import xarray as xr
+from openeo_gfmap.inference.model_inference import ModelInference
+
+
+class CroplandClassifier(ModelInference):
+    import functools
+
+    CATBOOST_PATH = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal-minimal-inference/wc_catboost.onnx"  # NOQA
+    BASE_URL = "https://s3.waw3-1.cloudferro.com/swift/v1/project_dependencies"  # NOQA
+    DEPENDENCY_NAME = "wc_presto_onnx_dependencies.zip"
+
+    def __init__(self):
+        import logging
+        logging.basicConfig(level=logging.INFO)
+        self.logger = logging.getLogger(WorldCerealInference.__name__)
+
+    @classmethod
+    @functools.lru_cache(maxsize=6)
+    def extract_dependencies(cls, base_url: str, dependency_name: str):
```

Comment:
Same comment as before, can't we take care of this in GFMAP and shorten the project-specific class?